understanding/21/pointer-cancellation.html

```diff
@@ -11,8 +11,9 @@ <h1>Understanding Pointer Cancellation</h1>
 		<section id="brief">
 			<h2>In brief</h2>
 			<dl>
-				<dt>Objective</dt><dd>Reduce accidental activation of controls</dd>
-				<dt>Author task</dt><dd>Make pointer cancellation predictable and consistent</dd>
+				<dt>Goal</dt><dd>Reduce accidental activation of controls.</dd>
```

Comment:
I think it might help to add in the main means of operating pointers...
```suggestion
				<dt>Goal</dt><dd>Reduce accidental activation of controls by mouse or touch.</dd>
```